pkg/tablewriter/protobuf.go

```diff
@@ -68,6 +69,18 @@ func (table *ProtobufWriter) Flush() error {
 	return nil
 }
 
+// FlushWithError flushes the buffer and end with the error message
+func (table *ProtobufWriter) FlushWithError(e error) error {
+	if e := table.Flush(); e != nil {
```

Comment:
Tiny suggestion. It's better to rename the input parameter `e` so as to separate the other error.